src/linagora.esn.contact/app/contact/form/contact-edition-form.directive.spec.js

```diff
@@ -1,29 +1,57 @@
 'use strict';
 
-/* global chai: false */
+/* global chai, sinon: false */
 
 var expect = chai.expect;
 
-describe('The contactEditionForm directive', function() {
-  var $compile, $rootScope, $scope;
+describe.only('The contactEditionForm directive', function() {
```

Comment:
Forgot to remove the .only
```suggestion
describe('The contactEditionForm directive', function() {
```

Reply:
thanks good catch